scripts/generate_contract_param_methods.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
```

Comment:
If you want to leave this in lemme know so that we can have a call about code-generation in rust